debian/vzlogger.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=vzlogger
+After=syslog.target network.target ntp.service
+
+[Service]
+ExecStart=/usr/bin/vzlogger -c /etc/vzlogger.conf
```

Comment:
i think we wsnt to use the foreground option for systemd, see https://github.com/volkszaehler/vzlogger/pull/453

Reply:
No, we don't want to.
Foreground is for command line (or scripts). Standard usage without option is background as a daemon.

Reply:
in the systemd-world, services don't need to "daemonize" themselves, as systemd can manage processes better than init-scripts used to.
actually systemd has to do extra work to track the pid of processes as they try to detach themselves, just so just can maintain control of them.
https://www.freedesktop.org/software/systemd/man/systemd.service.html
i.e we could do: Type=exec ExitType=main